Russian translation

kemika1girl opened the pull request:
There is still a lot of work left to do, but it's better than nothing! i will try to translate more, if i find free time for this

Commit:
Update some part of ru_ru.json

Commit:
Update ru_ru.json

Extegral commented:
Oh, finally someone other than me interested in updating that localization.

I'd want to point out a couple things about that translation of yours, and it would be simpler to continue conversation in Russian, assuming we both know it well enough. Would you mind?

kemika1girl commented:
да, конечно, можно продолжить на русском! уточняйте всё, что считаете нужным; буду только рада, если вы подправите некоторые названия так, как считаете нужным, ибо я сомневаюсь насчет собственного перевода некоторых (вроде the twist)

Commit:
Update ru_ru.json

Commit:
Update ru_ru.json

Commit:
Update ru_ru.json

Commit:
Update ru_ru.json

Aizistral reviewed (changes requested):
Хорошо. Ну, во-первых, файл `ru_ru[OUTDATED].json` содержит локализацию, которой некогда сам занимался, оставив по причине нехватки времени и желания на её поддержание. Я был бы признателен, если бы по возможности вы ориентировались на него при переводе названий предметов и тултипов, которые уже переведены в старом файле.

И да, есть несколько спорных решений в ваших трудах. "Усиленное магнетическое кольцо", конечно, передаёт смысл, но есть причины почему я не назвал его так в английской локализации, поэтому хотелось бы видеть более точный перевод, что-то вроде "Кольцо дислокации", ну или "Кольцо смещения", как я некогда окрестил его в старом переводе.

"Нечистый грааль" это конечно приемлемый перевод, но по мне "Осквернённый грааль" звучит более впечатляюще.

"Суперабсорбирующую губку" вполне можно было дословно перевести как "Экстраполированную мега-губку". Хотя тут не так уверен.

Почему именно "Божье благословение"? Чем "ангельское" по душе не пришлось?

"Ящик" можно перевести как "Один Ящик", так как его оригинальное название - отсылка к Властелину Колец.

"Эфирная руда" и все связанные предметы я бы предпочёл с названием "Эфириумный", в соответствии с оригинальным "Etherium" - что не совсем то же самое, что просто "Ether" или "Etheric".

"Grace of the Creator" переводится скорее как "Благословение творца", нежели "Изящество".

"Чернила Писаря" - тут скорее "Чернильница Писаря", ну или "Чернильница Архитектора", если уж мы хотим соответствия с оригиналом.

"Алчный пакт" можно и поэффектнее назвать. В соответствии с оригинальным именем - почему бы не "Пакт безграничной алчности".

Ну и да, "Обман" - это совсем не оно. В оригинале имеется ввиду что-то ближе к "Поворот". Ну, типа, "Вот это поворот!". В русском конечно одним словом так просто не передашь. Можно назвать "Сюжетный поворот" или "Неожиданный поворот", из вариантов.

Под "Curse of Nemesis" имеется ввиду что-то ближе к "Проклятие возмездия".

Конкретно по адвансментам, тут небольшая претензия только к фразе "Найдите или изготовьте". Мне кажется "Найдите или создайте" лучше подойдёт.

Стандартный для таких случаев совет - не забывайте проверять отображение вашей локализации непосредственно в игре, думаю знаете как, если не первый раз занимаетесь. Конкретно это важно для тултипов, поскольку они вручную поделены на строки, поэтому важно убедиться, что разные строк в пределах тултипа одного предмета не имеют большой разницы в длинне. Также важно для названий глав и самих глав Откровения (вот с этим названием вы уловили суть, кстати), поскольку те и другие вмещают в себя ограниченное количество текста.

Ну и да, в любом случае, я бескрайне благодарен за ваше участие.

kemika1girl commented:
хорошо, я учту всё это и сейчас подправлю, но хочу тоже сделать несколько замечаний:
1. губка
насколько мне известно, вы вдохновлялись террарией, разрабатывая этот мод, потому я смею предположить, что на создание этой губки вас натолкнула та самая губка из террарии, которая именуется "суперабсорбирующая губка";
2. кольцо дислокации
под дислокацией скорее имеется ввиду именно смещение какого-то тела с какой-то позиции, но не притягивание (это, конечно, каждый может понимать по-разному, так что если уж очень сильно хочется видеть именно "дислокацию" - скажите снова)

Commit:
Update ru_ru.json

Данная версия перевода завершена где-то на 50% и полностью проверена в игре!

Commit:
Update ru_ru.json

Extegral commented:
Ну, на самом деле конкретно мега-губка была независимым от террарии изобретением, и я только сейчас осознал, что её функционал схож с суперабсорбирующей губкой в террарии. Но в таком случае можно окрестить и эту суперабсорбирующей, в качестве ещё одной отсылки.

Насчёт кольца - нюанс в том, что Ring of Dislocation не притягивает предметы, и даже не телепортирует к игроку. На уровне кода предметы в радиусе его действия ведут себя так, как будто игрок уже стоит достаточно близко к ним, чтобы поднять естественным образом. То есть, это даже не телепортация предметов к игроку, а скорее как телепортация игрока к предметам. Этот технический момент показался мне довольно забавным, поэтому я отчасти передал его как в лоре, так и в самом названии кольца.

kemika1girl commented:
вот как! тогда да, название оправдывает себя

Commit:
Update ru_ru.json

подправлена подсказка откровения и свитка посмертного возврата

kemika1girl commented:
здравствуйте! возникла небольшая проблема перевода этих двух строк:
`"tooltip.enigmaticlegacy.theTwist1": "On a path to take it all away,",`
`"tooltip.enigmaticlegacy.theTwist2": "there can be no better way of knowing...",`
хотелось бы узнать конкретно, какой смысл вкладывался в эти слова

Commit:
Update ru_ru.json

полный перевод всех предметов и их описаний;
осталось перевести лишь содержание книги

Extegral commented:
> здравствуйте! возникла небольшая проблема c переводом этих двух строк:
> `"tooltip.enigmaticlegacy.theTwist1": "On a path to take it all away,",`
> `"tooltip.enigmaticlegacy.theTwist2": "there can be no better way of knowing...",`
> хотелось бы узнать конкретно, какой смысл вкладывался в эти слова

Эта пара строк процитирована из песни Disturbed - The Night. Можете оставить непереведённым для сохранения отсылки, или написать что-то оригинальное заместо.

Extegral commented:
И как там успехи с содержанием книги?

kemika1girl commented:
всё ещё в процессе, так как я приостановила перевод по причине возобновления учёбы, но я постараюсь как-нибудь закончить в ближайшее время

Extegral commented:
По некоторым размышлениям я пришёл к заключению, что какой-то перевод будет лучше никакого, поэтому волью это сейчас. Откройте новый PR если будет дальнейший прогресс с локализацией.
